pkg/mermaid/integrationdiagram/integrationdiagram.go

```diff
@@ -15,12 +15,30 @@ type integrationPair struct {
 }
 
 //GenerateIntegrationDiagram accepts an application name as input and returns a string (and an error if any)
-//The resulting string is the mermaid code for the integration diagram for that application
+//The resulting string is the mermaid code for the integration diagram
```

Comment:
Just looking for some clarification for this function, what's the reason behind wrapping `generateEntireIntegrationDiagramHelper ` and `generateIntegrationDiagramHelper ` in a single function here instead of having 
`GenerateEntireIntegrationDiagram` and `GenerateIntegrationDiagram` as two public functions?

It seems to me like it'd be easier for users to use whichever one is appropriate, rather than needing to know that passing an empty string into the appName param will generate a diagram for everything

Reply:
Yep, makes sense. I'll split them into two functions.